app/views/catalog/_field_list.html.erb

```diff
@@ -1,10 +1,22 @@
-<dl>
-  <% fields.each do |method| 
-    asset.send(method).tap do |value| 
-      if !value.empty? %>
-        <dt><%= method.gsub('_',' ').gsub(/clip /i, '') %></dt>
-        <dd><%= value %></dd><% 
-      end
-    end
-  end %>
-</dl>
\ No newline at end of file
+<%
+# default to 1 column, and only allow up to 4
+columns ||= 1
+columns = [columns.to_i, 4].min
+bootstrap_colspan = 12 / columns
+%>
+
+
+<div class="row field-list">
+  <% field_list.in_groups(columns) do |field_group| %>
+    <div class="col-md-<%= bootstrap_colspan %>">
+      <dl>
+        <% field_group.each do |field| %>
+          <% unless field.nil? %>
```

Comment:
[it will be nil by default](http://apidock.com/rails/Array/in_groups) when the modulo != 0. Alternatively, can pass false to `in_groups` to avoid it.